src/main/kotlin/no/nav/familie/ef/sak/økonomi/TilkjentYtelseController.kt

```diff
@@ -0,0 +1,64 @@
+package no.nav.familie.ef.sak.økonomi
+
+import no.nav.familie.ef.sak.sikkerhet.SikkerhetContext
+import no.nav.familie.ef.sak.økonomi.dto.TilkjentYtelseRestDTO
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+
+@RestController
```

Comment:
Antar at flere av endepunktene i denne controlleren er for praktisk testing inntil vi har mer kjøtt på beinet.

Reply:
Ja. Vil ha en dialog om hvordan API'et skal se ut rundt utbetaling.